openpype/hosts/maya/plugins/create/create_review.py

```diff
@@ -32,6 +38,30 @@ def __init__(self, *args, **kwargs):
         super(CreateReview, self).__init__(*args, **kwargs)
         data = OrderedDict(**self.data)
 
+        project_name = legacy_io.Session["AVALON_PROJECT"]
```

Comment:
We are trying to reduce the usage of legacy_io. Right @iLLiCiTiT 

This should be using `get_current_project()` from [here](https://github.com/ynput/OpenPype/blob/77f8a8ad0c36e2075a70132f963e8248f2a3407e/openpype/pipeline/context_tools.py#L300-L311)
```suggestion
        project_name = get_current_project()["name"]
```
It'd be much more optimal however if there was a `get_current_project_name` which didn't require the database query. I thought there was a function like that but I couldn't find it.

Reply:
There is actually function to get only project name:
```suggestion
        project_name = get_current_project_name()
```

```python
# Import
from openpype.pipeline import get_current_project_name
```